promptsource/templates/jigsaw_unintended_bias/templates.yaml

```diff
@@ -0,0 +1,212 @@
+dataset: jigsaw_unintended_bias
+templates:
+  17f730bd-fc7e-419c-84cc-e5c795268d02: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: 17f730bd-fc7e-419c-84cc-e5c795268d02
+    jinja: 'Online comments can sometimes be toxic. For instance, they might be insulting, threatening, racist or
+    many other issues. If 0 corresponds to a very civil comment, and 100 to a highly toxic one, how would you grade the
+    following comment?
+
+    "{{comment_text}}"
+
+    |||
+
+    {{(target * 100.0) | round(3) }}
```

Comment:
need to wrap conditionally the target since it is not present in the test splits (is it?)

Reply:
This is not an issue for Jigsaw, since after the kaggle competition the target was released for both the public and private sets. These splits are included in the version I pushed to `datasets`. 

What might be an issue is the use of unconventional split names. There is no proper validation set and there are two test sets, neither of which are named just "test".